lib/search_config.rb

```diff
@@ -10,6 +10,8 @@
 require "erb"
 
 class SearchConfig
+  ParseError = BaseParameterParser::ParseError
```

Comment:
I see you changed where this was used to use `BaseParameterParser::ParseError` but think you might have forgotten to delete this line. :)